spras/config.py

```diff
@@ -222,16 +223,36 @@ def process_config(self, raw_config):
                 if cur_params[run_params]:
                     for p in cur_params[run_params]:
                         param_name_list.append(p)
-                        all_runs.append(eval(str(cur_params[run_params][p])))
+                        obj = str(cur_params[run_params][p])
+                        try:
+                            obj = [int(obj)]
+                        except ValueError:
+                            try:
+                                obj = [float(obj)]
+                            except ValueError:
```

Comment:
Should we also check for `bool` along with `int` and `float` before moving to these ranges?